ghost/core/core/server/services/members/api.js

```
@@ -75,130 +75,145 @@ function createApiInstance(config) {
-                        ${t('Sent to {email}', {email})}
-                        ${t('If you did not make this request, you can simply delete this message.')} ${t('This email address will not be used.')}
-                        `;
-                case 'signin':
-                default:
-                    return trimLeadingWhitespace`
-                        ${t(`Hey there,`)}
+                            ${t('Sent to {email}', {email})}
+                            ${t('If you did not make this request, you can simply delete this message.')} ${t('This email address will not be used.')}
+                            `;
+                    case 'signin':
+                    default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)